includes/amp-helper-functions.php

```diff
@@ -929,9 +929,20 @@ function amp_get_content_sanitizers( $post = null ) {
 	}
 
 	if ( $amp_to_amp_linking_enabled ) {
```

Comment:
Let's add a check here to make sure this conditional never is entered when in Standard mode.

```suggestion
	if ( $amp_to_amp_linking_enabled && AMP_Theme_Support::STANDARD_MODE_SLUG !== AMP_Theme_Support::get_support_mode() ) {
```

AMP-to-AMP linking is only relevant in a paired AMP mode, that is Transitional and Reader.